StudyHub/StudyHub/Controllers/AssignmentController.cs

```diff
@@ -68,6 +68,15 @@ public async Task<IActionResult> UpsertStudentAnswer(StudentAnswerDTO dto)
         return result ? Ok() : BadRequest();
     }
 
+    [HttpGet("student-answer")]
+    [Authorize(Roles = "Teacher")]
+    public async Task<IActionResult> GetStudentAnswer(Guid variantId, Guid userId)
+    {
+        var result = await _studentAnswerService.GetStudentAnswerAsync(variantId, userId);
```

Comment:
As I understood this method of checking students' answers, so I think it is better to get answers by assignmentId

Reply:
1) we can't there is no props - assignmentId or assignment
2) we have many variants for one assignment, but we need the only one
so, there is no need to change